api/route/route.go

```diff
@@ -209,6 +223,15 @@ func updateRoute(c *gin.Context) {
 			}
 		}()
 		logger.Info(rd)
+		if isCreateGroup {
+			if err := tx.Model(&service.RouteGroupDao{}).Create(routeGroup).Error; err != nil {
+				tx.Rollback()
```

Comment:
why we need a transaction here ?

Reply:
@liuxiran Please take look :D

Reply:
Refer to the original design, route must be in a group, if the group has  not exited, create with the route. so here if the group create with error, do rollback :) @nic-chen

Reply:
@liuxiran  Sorry, it looks like a separate transaction.